### ISSUE TITLE
v0.17.9 for MacOS is not signed

### ISSUE THREAD

**Issue by freguinhas** [docker/kitematic#5370]
### Expected behavior
App should open normally and without any alerts on MacOS

### Actual behavior
MacOS displays a message informing that the OS cannot verify the app's developer and stops the user from opening it.

### Information about the Issue
The App must be signed using an Apple Developer certificate.

```
$ spctl -a -v -v /Applications/Kitematic.app
/Applications/Kitematic.app: rejected
source=no usable signature
```

```
$ codesign -v /Applications/Kitematic.app
/Applications/Kitematic.app: code object is not signed at all
In architecture: x86_64
```

### Steps to reproduce the behavior

1. Download from GitHub (https://github.com/docker/kitematic/releases/download/v0.17.9/Kitematic-0.17.9-Mac.zip)
2. Open on macOS Catalina

**Comment by Trackhe** [comment]
Can you open that file with shift? Hold shift then open.

**Comment by JasinYip** [comment]
I'm facing the same issue. Can't use it now.

**Comment by Trackhe** [comment]
Sorry i mean hold STRG and open it!
![Bildschirmfoto 2019-10-23 um 15 51 31](https://user-images.githubusercontent.com/1494328/67399758-fed40a80-f5ac-11e9-9d7b-352061a9992e.png)

**Comment by arekdist** [comment]
I confirm the same. Kitematic-0.17.9 is not signed and notarized.

**Comment by freguinhas** [comment]
@Trackhe that is a system bypass and should not be recommended.

**Comment by Trackhe** [comment]
yes okey but this is an opensource free project. you can pay or making self your sigend version.
here are the new requirements. https://developer.apple.com/documentation/security/notarizing_your_app_before_distribution?language=objc

**Comment by schrej** [comment]
Docker Inc. recommends Kitematic and offers it as a download. They should have the resources to sign it, especially considering that they also provide Docker for Mac, which is signed.
I understand that a lot of free projects can't (or don't want to - which is also fine) afford a Apple developer certificate. But Kitematic is supported by Docker and advertised as "by Docker" on https://kitematic.com - there should be a signed version available in my opinion.

One reason for example: On my corporate MacBook using Apps not signed with a valid certificate is forbidden for example, so I can't use Kitematic on it. And I guess other companies might have the same requirements as well.

**Comment by Trackhe** [comment]
I would also be glad to hear about better support kitematic.

**Comment by schrej** [comment]
That's no reason to not provide signed versions of Kitematic until the Dashboard reaches feature parity. If they have something new to replace it that is ready to use, I'm fine with switching, but until then, a signed version should be provided.

**Comment by apeeters** [comment]
0.17.10 is also not signed. Latest signed version is 0.17.8.

**Comment by Trackhe** [comment]
spend 99usd and get your sign version. ^^ maybe with dark-mode.

**Comment by ramumnesys** [comment]
Upvoting this issue. Come on guys, it's been hanging for 3 months already.

**Comment by fregadev** [comment]
I feel the need to reassure everyone here that @Trackhe is not in anyway associated with Kitematic or Docker. I am blocking him and strongly urge you all to do the same to any trolls you may encounter on Github.

**Comment by huhlig** [comment]
Problem still exists as of v0.17.11

**Comment by oguennec** [comment]
I am having the same issue with Kitematic v0.17.12 installed through `brew cask install kitematic`

**Comment by JasinYip** [comment]
It's been about 10 months and now [docker desktop](https://www.docker.com/products/docker-desktop) offers a dashboard so we can just use it instead of kitematic.
